tests/components/media_player/test_device_trigger.py

```diff
@@ -61,12 +62,13 @@ async def test_get_triggers(hass, device_reg, entity_reg):
     entity_reg.async_get_or_create(DOMAIN, "test", "5678", device_id=device_entry.id)
 
     trigger_types = {
-        "turned_on",
-        "turned_off",
+        "buffering",
```

Comment:
We're missing a translation string update.

https://github.com/home-assistant/core/blob/e6b88434230f51c8f07f6c2fa056ad3200190445/homeassistant/components/media_player/strings.json#L11-L18

Reply:
Added to #70863